leverage/_utils.py

```
@@ -112,6 +115,19 @@ def __init__(self, exit_code: int, error_description: str):
 
 
+def parse_tf_file(file: Path):
+    """
+    Open and parse an HCL file.
+    In case of a parsing error, raise a user-friendly error.
+    """
+    with open(file) as f:
+        try:
+            parsed = hcl2.load(f)
+        except lark.exceptions.UnexpectedInput:
+            raise ExitError(1, f"There is a parsing error with the {f.name} file. Please review it.")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)